src/main.py

```diff
@@ -652,39 +644,23 @@ def run(settings, components, variables, pyomoChk=False):
   outputType = settings.getOutput()
 
 
-  some_data = {**projectCashflows, **compCashflows }
+  results = {}
```

Comment:
did you rebase or merge devel into this branch, or did you make changes manually?